Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -55,6 +55,17 @@ class ContentBlockerHelper {
             }
         }
 
+        static func accessibilityId(for state: EnabledState) -> String {
+            switch state {
+            case .on:
+                return "Settings.TrackingProtectionOption.OnLabel"
```

Comment:
IMO this should be `Strings.trackingProtectionAccessibilityId.onLabel`, otherwise this string gets repeated in 3 places, 1) Strings.swift, 2) here, 3) when used in tests.
But there is no precedent in the code for this method.

Then tests would be able to include Strings.swift, and use constants for accessibility identifiers.

Reply:
I'd like to come up with some standard for `accessibilityId`.